enterprise/backend/src/metabase_enterprise/serialization/v2/extract.clj

```diff
@@ -56,16 +57,16 @@
                     (serdes.base/extract-all model opts)))))
 
 ;; TODO Properly support "continue" - it should be contagious. Eg. a Dashboard with an illegal Card gets excluded too.
-(defn- descendants-closure [_opts targets]
```

Comment:
Sorry to butcher this one with bikeshedding... just seems a little more grokkable with "found" vs. "chased."